backend/src/pilstark/json_exporter/mod.rs

```diff
@@ -173,6 +173,18 @@ fn polynomial_reference_type_to_type(t: &str) -> &'static str {
     }
 }
 
+/// Makes names compatible with estark, which sometimes require that
+/// there is exactly one `.` in the name.
+fn fixup_name(name: &str) -> String {
+    if name.contains('.') {
+        name.to_string()
+    } else if let Some(last) = name.rfind("::") {
+        format!("{}.{}", &name[..last], &name[last + 1..])
+    } else {
+        format!("Global.{name}")
```

Comment:
is this safe?

Reply:
I'll check when this actually happens.

Reply:
I think it should be fine. It's only called for witness or intermediate columns and they should be in a namespace. Changed it to a panic.